__init__.py

```diff
@@ -363,13 +365,16 @@ def _decrypt(self, value, encrypted_value):
         if value or (encrypted_value[:3] not in [b'v11', b'v10']):
             return value
 
-        # Encrypted cookies should be prefixed with 'v10' according to the
-        # Chromium code. Strip it off.
+        # Encrypted cookies should be prefixed with 'v10' or 'v11'.
+        # Choose key based on this prefix.
+        # Reference in chromium code: `OSCryptImpl::DecryptString` in
+        # components/os_crypt/os_crypt_linux.cc
+        key = self.v11_key if encrypted_value[:3] == b'v11' else self.v10_key
```

Comment:
As I mentioned earlier. `self.v11_key` and `self.v10_key` are only defined on Linux. This will cause trouble in Mac or Windows.